lib/logflare_web/live/endpoints/endpoints_live.ex

```diff
@@ -190,24 +178,18 @@ defmodule LogflareWeb.EndpointsLive do
   def handle_event("apply-beta", _params, %{assigns: %{user: user}} = socket) do
     Logger.debug("Endpoints application submitted.", %{user: %{id: user.id, email: user.email}})
 
-    {:noreply,
-     socket
-     |> put_flash(:info, "Successfully applied for the Endpoints beta. We'll be in touch!")}
+    message = "Successfully applied for the Endpoints beta. We'll be in touch!"
+    {:noreply, put_flash(socket, :info, message)}
   end
 
-  defp update_params_form(socket, parameters, query_string) do
+  defp assign_updated_params_form(socket, parameters, query_string) do
+    params = for(k <- parameters, do: {k, nil}, into: %{})
```

Comment:
this would still clear the form values, i think.

Reply:
No, all good 👍 works as expected gifs in the PR